src/Common/src/System/IO/PathInternal.Windows.cs

```diff
@@ -439,6 +414,7 @@ internal static int PathStartSkip(string path)
         /// <summary>
         /// True if the given character is a directory separator.
         /// </summary>
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
On a release build?  Into which function wasn't it being inlined?  I'd like to take a look to understand why.